cohort_4/playbook/activities.md

```diff
@@ -24,6 +29,51 @@
 
 TODO
 
+## Brainstorming
+
+### Forming groups
+
+- Assign people to groups.
+  - There are social pressures between friends when forming groups. Start by
+    assigning the hackers to random groups.
+  - If any problems arise in a group handle it on a case-by-case basis.
+
+### Yes-and sessions
+
+- In a "Yes-and" session, pairs of hackers build off each-others' ideas to create
+  long, completely nonsensical ideas
+  - "Yes-and" sessions are exercises that help people get out of a judgmental
+    mindset before brainstorming
+- To run a "Yes-and" session, start with an example between facilitators
```

Comment:
Down for that, I found this one but it's very different from what we've been doing (which seems to work well enough)

https://www.youtube.com/watch?v=NWN7j8A-lmk